figures/generate-figures.sh

```diff
@@ -182,6 +182,32 @@ Rscript --vanilla scripts/figS6-subtype-umap-panels.R
 # Panels E and F were previously generated with `scripts/fig5_figS6-immune-deconv-panels.R` in a Figure 5 step
 
 
+##### Figure S7: Transcriptomics results for only high tumor-purity samples -------------
+
+
+# Run script to generate TP53 and EXTEND score boxplots and associated legend
+Rscript --vanilla scripts/figS7-tp53-telomerase-tumor-purity-threshold.R
+
+
+# Copy panels for this plot (specific panel order is TBD):
+
+# TP53 scores across TP53 status
+cp ${analyses_dir}/tp53_nf1_score/results/tumor-purity-threshold/tp53-score-status_tumor-purity-threshold.pdf pdfs/supp/figs7/panels/
+
+# TP53 expression across TP53 status
+cp ${analyses_dir}/tp53_nf1_score/results/tumor-purity-threshold/tp53-fpkm-status_tumor-purity-threshold.pdf pdfs/supp/figs7/panels/
+
+# TP53 classifier ROC curve
+cp ${analyses_dir}/tp53_nf1_score/results/tumor-purity-threshold/tp53-roc_tumor-purity-threshold.pdf pdfs/supp/figs7/panels/
+
+# quanTIseq fractions across cancer groups
+cp ${analyses_dir}/immune-deconv/plots/tumor-purity-threshold_quantiseq-cancer-groups.pdf pdfs/supp/figs7/panels/
+
+# UMAP highlighting broad histologies
+cp ${analyses_dir}/transcriptomic-dimension-reduction/plots/umap_tumor-purity-threshold.pdf pdfs/supp/figs7/panels/
+
```

Comment:
Have all of these been adequately accounted for in `scripts/run-manuscript-analyses.sh`? That script hasn't changed since August, and I know you added a separate shell script at least for the TP53 classifier (`analyses/tp53_nf1_score/run_classifier-tumor_purity_threshold.sh`) since then. Should probably add `tumor-purity-exploration` as well. So, I've answer my own question and now suppose the question is what else needs to make it back in there?  New 🎫 required.